Makefile

```diff
@@ -222,3 +235,26 @@ swagger-docs: ## preview the API documentation
 		-e 'REDOC_OPTIONS=hide-hostname="true" lazy-rendering' \
 		-p $(SWAGGER_DOCS_PORT):80 \
 		bfirsh/redoc:1.6.2
+
+.PHONY: buildx
+ifeq ($(BUILDX), bundles/buildx)
+buildx: bundles/buildx # build buildx cli tool
+else
+buildx:
+endif
+
+bundles/buildx: GOOS ?= $(shell go env GOHOSTOS || uname | awk '{print tolower($$0)}' || echo windows)
+bundles/buildx: GOARCH ?= $(shell go env GOHOSTARCH || echo amd64)
+bundles/buildx: DOCKER_BUILDX_VERSION ?= v0.3.0
+bundles/buildx: DOCKER_BUILDX_REPO ?= https://github.com/docker/buildx.git
+bundles/buildx: BUILDX_DOCKERFILE ?= Dockerfile.buildx
+bundles/buildx: bundles ## build buildx CLI tool
+	# This intetionally is not using the `--output` flag from the docker CLI which is a buildkit option
+	# The idea here being that if buildx is being used, it's because buildkit is not supported natively
+	docker build -f $(BUILDX_DOCKERFILE) -t "moby-buildx:$(DOCKER_BUILDX_VERSION)" \
```

Comment:
I wonder if we have practical examples to override the `BUILDX_DOCKERFILE` (perhaps just hard-code `Dockerfile.buildx`)

Reply:
We support it elsewhere 🤷‍♂